src/app/@ansyn/imagery/model/base-imagery-visualizer.ts

```diff
@@ -8,15 +8,21 @@ export type VisualizerInteractionTypes =
 	| 'doubleClick'
 	| 'contextMenu'
 	| 'drawInteractionHandler'
-	| 'translateInteractionHandler';
+	| 'translateInteractionHandler'
+	| 'labelTranslateHandler'
+	| 'editAnnotationTranslateHandler'
+	| 'modifyInteractionHandler';
 
 export const VisualizerInteractions: { [key: string]: VisualizerInteractionTypes } = {
 	click: 'click',
 	pointerMove: 'pointerMove',
 	doubleClick: 'doubleClick',
 	contextMenu: 'contextMenu',
 	drawInteractionHandler: 'drawInteractionHandler',
-	translateInteractionHandler: 'translateInteractionHandler'
+	translateInteractionHandler: 'translateInteractionHandler',
```

Comment:
add remarks 
// translateInteractionHandler -> like change pixels offset  (for display only)
// editAnnotationTranslateHandler-> like change actual annotation position